protocols/protocol/cloudstate/entity.proto

```diff
@@ -69,41 +109,70 @@ message SideEffect {
     // The payload of the command.
     google.protobuf.Any payload = 3;
 
-    // Whether this side effect should be performed synchronously, ie, before the reply is eventually
-    // sent, or not.
+    // Whether this side effect should be performed synchronously, ie, before the reply is eventually sent, or not.
     bool synchronous = 4;
 }
 
-// A command. For each command received, a reply must be sent with a matching command id.
+//Command represents TODO
+//
+//---
+//
+//`Specification`:
+//
+//  - An `Command` is considered to be valid iff:
+//    - `entity_id` TODO
+//    - `id` TODO
+//    - `name` TODO
+//    - `payload` TODO
+//    - `streamed` TODO
 message Command {
 
     // The ID of the entity.
     string entity_id = 1;
```

Comment:
Why does `Command` carry the `entity_id` where an `EventSourcedEvent` does not (or vice-versa)? The `entity_id` is established through the `EventSourcedInit` message and seems to be stable for a `EventSourcedStreamIn` session.

Reply:
@marcellanz I believe this is due to the fact that Commands get redistributed amongst the CloudState Proxies based on the entity_key. @jroper should be able to sanity-check my response :)

Reply:
I agree with Marcel that the entity_id should be stable to the all EventSourcedStreamIn

Reply:
Pinging @jroper for historical context :)

Reply:
Do we document somewhere if we don't support client-side streaming RPCs?
I've re-read the CRDT spec, and I think we don't, yes?